packages/jest-haste-map/src/__tests__/index.test.js

```diff
@@ -532,9 +537,9 @@ describe('HasteMap', () => {
 
     return new HasteMap({mocksPattern: '__mocks__', ...defaultConfig})
       .build()
-      .then(({__hasteMapForTest: data}) => {
-        expect(console.warn.mock.calls[0][0]).toMatchSnapshot();
-      });
+      .catch(({__hasteMapForTest: data}) => {
```

Comment:
should we add a `expect.hasAssertions()` in case this does not throw and we get a false positive in the test?

Reply:
Good catch!